backend/pkg/api/data_access/vdb_withdrawals.go

```diff
@@ -393,12 +408,28 @@ func (d *DataAccessService) getNextWithdrawalRow(queryValidators []t.VDBValidato
 		withdrawalAmount = 0
 	}
 
+	ens_name, err := db.GetEnsNameForAddress(*address, utils.SlotToTime(nextWithdrawalSlot))
+	if err != sql.ErrNoRows {
+		return nil, err
+	}
+
+	contractStatusReq := []db.ContractInteractionAtRequest{{
+		Address: address.String(),
```

Comment:
**question:** I am not sure whether `address.String()` returns the string with the "0x" prefix, please check.

Reply:
It does, also it's the checksum hashed address which would not match